data_store.rb

```diff
@@ -0,0 +1,42 @@
+require 'json'
+require_relative 'music_album'
+require_relative 'genre'
+
+module DataStore
+  def load_albums
+    if File.exist?('album.json')
+      JSON.parse(File.read('album.json')).map do |album|
+        MusicAlbum.new(album['name'], album['genre'], album['label'], album['publish_date'], album['on_spotify'])
```

Comment:
Should remove Label from the parameters and add id in the beginning 
`MusicAlbum.new(album['id'],album['name'], album['genre'], album['publish_date'], album['on_spotify'])`